src/ui/Templates/DefaultResultTemplate.ts

```diff
@@ -23,30 +25,33 @@ export class DefaultResultTemplate extends Template {
     super();
   }
 
-  instantiateToString(queryResult: IQueryResult, instantiateOptions: IInstantiateTemplateOptions = {}): string {
-    Assert.exists(queryResult);
-    let merged = new DefaultInstantiateTemplateOptions().merge(instantiateOptions);
-    queryResult = _.extend({}, queryResult, UnderscoreTemplate.templateHelpers);
+  instantiateToString(object: IQueryResult | ITemplateMetaFields, instantiateOptions: IInstantiateTemplateOptions = {}): string {
```

Comment:
Mehhh... This class is getting pretty complicated.

Since the table layout is pretty different from the normal "default result template", wouldn't it make sense to have something dedicated for this ?

Instead of modifying this class so that it can support both code path ?